packages/itwinui-react/src/core/Buttons/SplitButton/SplitButton.tsx

```diff
@@ -28,6 +28,14 @@ export type SplitButtonProps = ButtonProps & {
    * Content of primary button.
    */
   children: React.ReactNode;
+  /**
+   * Passes props to SplitButton wrapper.
+   */
+  wrapperProps?: React.ComponentProps<typeof Box>;
+  /**
+   * Passes props to SplitButton menu button.
+   */
+  menuButtonProps?: React.ComponentProps<'button'>;
```

Comment:
this one should use icon button props
```suggestion
  menuButtonProps?: React.ComponentProps<typeof IconButton>;
```